Application/EdFi.Ods.AdminApi/Features/RequestLoggingMiddleware.cs

```diff
@@ -56,10 +56,12 @@ public async Task Invoke(HttpContext context, ILogger<RequestLoggingMiddleware>
                             validationResponse.errors[x.PropertyName] = new List<string>();
                         }
                         validationResponse.errors[x.PropertyName].Add(x.ErrorMessage.Replace("\u0027", "'"));
-                    });
-
-                    logger.LogDebug(JsonSerializer.Serialize(new { message = validationResponse, traceId = context.TraceIdentifier }));
-
+                    });
+
+#pragma warning disable S6667 // Logging in a catch clause should pass the caught exception as a parameter.
+                    logger.LogDebug(JsonSerializer.Serialize(new { message = validationResponse, traceId = context.TraceIdentifier }));
+#pragma warning restore S6667 // Logging in a catch clause should pass the caught exception as a parameter.

```

Comment:
I think it's enough to disable the warning

Reply:
I agree. I was about to ask you to write a reason into a code comment, but after reading the code, I think it is sufficiently self-explanatory. No extra comment needed.